account-kit/infra/src/actions/types.ts

```diff
@@ -60,7 +60,7 @@ export type RequestGasAndPaymasterAndDataRequest = [
     erc20Context?: {
       tokenAddress: Address;
       permit?: Hex;
-      maxTokenAmount?: number;
+      maxTokenAmount?: BigInt;
```

Comment:
is this meant to be BigInt and the one below bingint? What's the difference lol

Reply:
TBH they are the same. just BigInt is the interface and bigInt is the type :(